network-api/networkapi/wagtailpages/donation_modal.py

```diff
@@ -35,6 +38,25 @@ class DonationModal(TranslatableMixin, models.Model):
         default="Yes, I'll chip in",
     )
 
+    donate_url = models.CharField(
+        max_length=255,
+        default="?form=donate",
+        validators=[
+            RegexValidator(
+                regex=url_or_query_regex,
+                message=(
+                    "Please enter a valid URL (starting with http:// or https://), "
+                    "or a valid query string starting with ? (Ex: ?form=donate)."
+                ),
+            ),
+        ],
+        help_text=(
+            "If you would like this modal's donate button to link to a custom URL, "
```

Comment:
Hi @mmmavis, here is the updated help text! It now mentions that the url is for the modal's donate button